eamena/eamena/models/forms.py

```diff
@@ -854,11 +893,16 @@ def get_info():
 
     def update(self, data, files):
 
-        #if self.resource.entitytypeid not in ['ACTOR.E39']:
         self.update_nodes('GEOMETRIC_PLACE_EXPRESSION.SP5', data)
         self.update_nodes('GEOMETRY_EXTENT_CERTAINTY.I6', data)
         self.update_nodes('GRID_ID.E42', data)
-        self.update_nodes('SITE_OVERALL_SHAPE_TYPE_NEW.E55', data)
+        
+        if self.resource.entitytypeid == 'HERITAGE_FEATURE.E24':
+            siteshape_node = 'SITE_OVERALL_SHAPE_TYPE.E55'
+        else:
+            siteshape_node = 'SITE_OVERALL_SHAPE_TYPE_NEW.E55'
+        self.update_nodes(siteshape_node, data)
```

Comment:
@mradamcox  the _NEW tag will disappear at the end of the data migration process (using the rename_entity_type utility: https://github.com/azerbini/eamena_dev/blob/15561846fab51b0a8dc89aebd3240d8e41482c26/arches/management/commands/package_utils/migrate_resources.py#L385) so this bit of code will become redundant then.